From: Robert Schwebel <r.schwebel@pengutronix.de>

libtool 2.2.6a claims about these updates:

libtoolize: Consider adding `AC_CONFIG_MACRO_DIR([m4])' to configure.ac and
libtoolize: rerunning libtoolize, to keep the correct libtool macros in-tree.
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
# 20091228 rsc: mailed to radu.voicilas@gmail.com

 Makefile.am  |    1 +
 configure.ac |    1 +
 2 files changed, 2 insertions(+)

Index: inotify-tools-3.13/configure.ac
===================================================================
--- inotify-tools-3.13.orig/configure.ac
+++ inotify-tools-3.13/configure.ac
@@ -7,6 +7,7 @@ AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([src/inotifywait.c])
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_HEADERS([libinotifytools/src/inotifytools/inotify.h])
+AC_CONFIG_MACRO_DIR([m4])
 AC_DEFINE([_GNU_SOURCE], [], [For a few GNU-specific functions])
 AC_PROG_MAKE_SET
 
Index: inotify-tools-3.13/Makefile.am
===================================================================
--- inotify-tools-3.13.orig/Makefile.am
+++ inotify-tools-3.13/Makefile.am
@@ -1,3 +1,4 @@
+ACLOCAL_AMFLAGS = -I m4
 
 SUBDIRS = libinotifytools src man
 
